### ISSUE TITLE
Output size 1/8th of input

### ISSUE THREAD

**Issue by gfkri** [XuJiacong/PIDNet#115]
Hi,

thank you for your great work. Quick question, is it true that the output size of your network is 1/8 of the input resolution? So there is no more spatial information than this at the output? Only bilinear upsampling (on the logits) is used to get to the original size, right?

Thank you very much in advance, 
BR
Georg
